src/pivot.coffee

```diff
@@ -584,6 +590,40 @@ callWithJQuery ($) ->
     ###
 
     pivotTableRenderer = (pivotData, opts, rendererType) ->
+        #If we've been asked to generate the totals from a meta-aggregator, walk the data tree here and do that
+        if pivotData.opts.totalsMetaAggregator
+            for own flatRowKey, row of pivotData.tree
+                for own flatColKey, aggregator of row
+                    for [totals, metaAggTotals, oppAttrs, flatKey, oppKey] in [
```

Comment:
Style: we use "dim" (short for "dimension") to refer generically to a row or column. Not sure what "opp" stands for. But it's probably clearest to say "attrs" and "key", or, prepend "dim" to each of these variables.